validators/Aadhar_IN.js

```diff
@@ -0,0 +1,15 @@
+const dry = require('drytypes')
+
+module.exports = dry.makeDryType((x) => {
+    if (typeof x !== 'string') {
```

Comment:
1. typeof check needs a number
2. after checking the number, Length should be verified

Reply:
3. typeof should be both number and string
4. if its a number then it needs to be converted in string in order to use `.match` so the function will accept both `string` and `number`